src/Microsoft.Dnx.ApplicationHost/Program.cs

```diff
@@ -229,19 +229,24 @@ private Task<int> ExecuteMain(DefaultHost host, string applicationName, string[]
             {
                 assembly = host.GetEntryPoint(applicationName);
             }
-            catch (FileNotFoundException ex) when (new AssemblyName(ex.FileName).Name == applicationName)
+            catch (Exception ex)
```

Comment:
Before we threw with `ThrowEntryPointNotfoundException` only if `ex.FileName == applicationName` now we will throw for any FileLoadException or FileNotFoundException - is it intentional?